comments/urls.py

```diff
@@ -1,17 +1,21 @@
 from __future__ import absolute_import, unicode_literals
-from django.conf.urls import url
 from django.urls import path
 
 from comments import views
-from comments.views import ProcessCannedResponseView, CommentsCommunityModerationView
+
 
 urlpatterns = [
-    url(r'^comment/(?P<comment_pk>\d+)/update/(?P<action>publish|unpublish|hide|show|clear_flags|approve|reject|unsure)/$',
-        views.update, name='wagtail_comments_xtd_publication'),
     path('comment/<int:comment_pk>/reply',
         views.CommentReplyView.as_view(), name='comment_reply_view'),
     path('comment/new',
             views.post_admin_comment, name='comment_post_admin_comment'),
-    path('comment/process-canned-response', ProcessCannedResponseView.as_view(), name='process_canned_response'),
-    path('community-moderation/', CommentsCommunityModerationView.as_view(), name='comments_community_moderation'),
+    path('comment/process-canned-response', views.ProcessCannedResponseView.as_view(), name='process_canned_response'),
+    path('community-moderation/', views.CommentsCommunityModerationView.as_view(), name='comments_community_moderation'),
+    path('comment/<int:comment_pk>/approve', views.ApproveCommentView.as_view(), name='approve_comment'),
+    path('comment/<int:comment_pk>/reject', views.RejectCommentView.as_view(), name='reject_comment'),
+    path('comment/<int:comment_pk>/unsure', views.UnSureCommentView.as_view(), name='unsure_comment'),
+    path('comment/<int:comment_pk>/hide', views.HideCommentView.as_view(), name='hide_comment'),
+    path('comment/<int:comment_pk>/publish', views.PublishCommentView.as_view(), name='publish_comment'),
+    path('comment/<int:comment_pk>/unpublish', views.UnPublishCommentView.as_view(), name='unpublish_comment'),
+    path('comment/<int:comment_pk>/clear-flags', views.ClearFlagsCommentView.as_view(), name='clear_flags'),
```

Comment:
I'd like to establish a convention for naming these paths. Should we prefix each one with the name of the app and/or model e.g. `comments_comment_approve` or `comment_approve`? Do we need to prefix at all? In any case, I'd like the naming to go from the general to the specific, rather than the other way around, as it is now.

You can leave the existing named paths as they are and focus just on the new ones we are introducing.

Reply:
I should say that I'm happy with the URL structure i.e. 'comment/\<id\>/\<action\>'.